contextualhelp/data.json

```diff
@@ -423,17 +454,27 @@
+        "content": "Qui puoi controllare e modificare le impostazioni utilizzate da tutti i servizi per personalizzare la tua esperienza su IO. Puoi anche selezionare il calendario su cui vuoi salvare i promemoria, abilitare l’inoltro automatico di un’anteprima dei messaggi che ricevi su IO al tuo indirizzo email e modificare la lingua dell’app.",
         "faqs": [
           {
             "title": "Come viene tutelata la mia privacy?",
-            "body": "L’app IO è stata concepita e sviluppata secondo i principi del [Regolamento generale sulla protezione dei dati](https://eur-lex.europa.eu/legal-content/IT/TXT/HTML/?uri=CELEX:32016R0679&from=EN), conosciuto anche come GDPR, e del [Codice privacy italiano](https://www.normattiva.it/uri-res/N2Ls?urn:nir:stato:decreto.legislativo:2003-06-30;196!vig=). È quindi uno strumento sicuro che protegge le tue informazioni e i tuoi diritti.\n\nPer maggiori informazioni, leggi l’Informativa sul trattamento dei dati personali alla voce 'Privacy e Condizioni d’uso'."
+            "body": "Abbiamo progettato e sviluppato l’app IO secondo i principi del [Regolamento generale sulla protezione dei dati](https://eur-lex.europa.eu/legal-content/IT/TXT/HTML/?uri=CELEX:32016R0679&from=EN), e del [Codice privacy italiano](https://www.normattiva.it/uri-res/N2Ls?urn:nir:stato:decreto.legislativo:2003-06-30;196!vig=)./n Proteggere le tue informazioni e i tuoi diritti è una nostra priorità.\n\nPer approfondire, leggi [l’Informativa sul trattamento dei dati personali](https://io.italia.it/app-content/tos_privacy.html)"
           },
           {
             "title": "Come posso eliminare il mio account e cancellare i miei dati?",
-            "body": "Vai alla voce 'Privacy e Condizioni d’uso' e seleziona 'Elimina il tuo account'. Al termine dell'eliminazione, riceverai un messaggio di conferma all’indirizzo email associato all'app. Questo non significa che non potrai più utilizzare IO: se accedi di nuovo, si genererà un nuovo account."
+            "body": "Puoi farlo da 'Privacy e Condizioni d’uso'e poi 'Elimina il tuo profilo. Se cambi idea e vuoi nuovamente usare IO, potrai sempre entrare di nuovo su IO tramite SPID o CIE."
           }
         ]
+      },      
+      {
+        "route_name": "PIN_SCREEN",
+        "title": "Codice di sblocco",
+        "content": "**Il codice di sblocco è il codice di sicurezza che mantiene al sicuro il tuo accesso all’app.**.\nSe è la prima volta che entri nell'app, imposta liberamente il tuo codice numerico e memorizzalo in modo da poterlo utilizzare durante tutti i successivi accessi. Se l'hai già impostato e non lo ricordi, puoi selezionare 'Hai dimenticato il codice di sblocco?'e impostare un nuovo codice.\nIl codice di sblocco è legato al dispositivo su cui l'hai impostato: se lo cambi, dovrai impostarlo nuovamente al primo accesso all'app IO."
+      },
+      {
+        "route_name": "ONBOARDING_TOS",
```

Comment:
This route's repeated twice per language. That's wrong. We should have only one route per language.